tests/test_uplift.py

```diff
@@ -28,11 +32,31 @@
 84.0a1
 """
 
+MILESTONE_TEST_CONTENTS_2 = """
+# Holds the current milestone.
+# Should be in the format of
+#
+#    x.x.x
+#    x.x.x.x
+#    x.x.x+
+#
+# Referenced by build/moz.configure/init.configure.
+# Hopefully I'll be able to automate replacement of *all*
+# hardcoded milestones in the tree from these two files.
+#--------------------------------------------------------
+
+105.0
+"""
+
 
-def test_parse_milestone_major_version():
-    assert (
-        parse_milestone_major_version(MILESTONE_TEST_CONTENTS) == 84
-    ), "Test milestone file should have 84 as major milestone version."
+def test_parse_milestone_version():
```

Comment:
Can test coverage be added to cover the not so happy path of `InvalidVersion`/`ValueError`?